internal/statestore/redis_indices.go

```diff
@@ -79,10 +81,30 @@ type indexFilter struct {
 	min, max float64
 }
 
-func extractIndexFilters(p *pb.Pool) []indexFilter {
+func extractIndexFilters(cfg config.View, p *pb.Pool) ([]indexFilter, error) {
+	indices := make(map[string]struct{})
+	if cfg.IsSet("ticketIndices") {
```

Comment:
If not set - should we just fail? or should we let the corresponding filter fail? Easier debugging if we tell the user - index not configured for Open Match?

Reply:
For simple low scale users, I think not having any indexed fields, and handling all logic in a single mmf is a reasonable scenario.